package.json

```diff
@@ -43,7 +43,8 @@
   },
   "dependencies": {
     "prop-types": "^15.5.8",
-    "reactify": "^1.1.1"
+    "reactify": "^1.1.1",
+    "require-dir": "0.3.2"
```

Comment:
Is this needed? If so, could it be in `devDependencies`?

Reply:
It was not needed, I have removed it.
I had forgotten to remove it before creating the PR.
Thanks.